### ISSUE TITLE
Project configuration don't load properly

### ISSUE THREAD

**Issue by antobenve** [fxi/map-x-mgl#304]
The popup window fails to load "title", "description" and map parameters.
![image](https://user-images.githubusercontent.com/30286565/44475720-11178900-a636-11e8-944f-c3bcff46e5d5.png)

**Console**
[test.app.mapx.org-1534954049546.log](https://github.com/fxi/map-x-mgl/files/2311117/test.app.mapx.org-1534954049546.log)

**Comment by fxi** [comment]
Have you tried to reload the app ?

**Comment by antobenve** [comment]
I'm sorry Fred. I though the content of the console was the best thing to send you.
However, I tried to reload the app several time without solving the problem.
Now, I actually managed to have the conf of the project to load properly.
I think this issue might be related to the #305 because the project I had these two issues is the same.
However, I really need to go now and I can't help you. I'll check better tomorrow morning if I find other similar problems in other projects as well and if this is solved.

**Comment by fxi** [comment]
Yep, sorry, I've missed the file. That's good.
So, it's should return an error and disable the button. 
To try again..

**Comment by thomaspiller** [comment]
FYI, I had a similar issue this morning with the popup window of the dashboard editor.

**Comment by fxi** [comment]
@thomaspiller, same errors in logs ? 
It happens all the time on both test.* and app.* ? Only once ? First time ? Randomly ?
Right after the service worker reload ?
service-workers.js had a no-cache header in both test.* and app.* ?

**Comment by fxi** [comment]
@thomaspiller Is this still happening ? Could you provide what I've asked or it's too technical ?

**Comment by thomaspiller** [comment]
@fxi Sorry, I had a busy day yesterday and I totally forgot to answer.
Unfortunately I was doing two things at the same time and I instinctively made a reload of the page and lost the log. Rookie mistake, sorry for that. 
This problem only happened once and was resolved when the page was reloaded. I was not able to reproduce it since. I tested on app & test on Chrome 68 and firefox 61 (Windows 10 Pro).
